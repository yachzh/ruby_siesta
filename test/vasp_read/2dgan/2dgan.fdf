SystemLabel   2dgan
NumberOfAtoms   2
NumberOfSpecies   2
LatticeConstant   1.0 Ang
%block LatticeVectors
     3.21800000      0.00000000      0.00000000
    -1.60900000      2.78686975      0.00000000
     0.00000000      0.00000000     40.00000000
%endblock LatticeVectors

AtomicCoordinatesFormat   Ang
%block ChemicalSpeciesLabel
  1   31   Ga
  2    7   N
%endblock ChemicalSpeciesLabel

%block AtomicCoordinatesAndAtomicSpecies
     0.00000000      0.00000000     20.00000000   1
     0.00000000      1.85791317     20.00000000   2
%endblock AtomicCoordinatesAndAtomicSpecies

PAO.BasisSize   DZP
PAO.EnergyShift   50 meV
WriteMullikenPop   1
WriteCoorXmol   true
DM.MixingWeight   0.01
DM.NumberPulay   5
DM.UseSaveDM   true
DM.Tolerance   0.0001
MaxSCFIterations   2000
SCF.H.Converge   true
SCF.H.Tolerance   1.0 meV
OccupationFunction   MP
OccupationMPOrder   4
XC.functional   GGA
XC.authors   PBE
ElectronicTemperature   1000 K
Mesh.Cutoff   200 Ry
Spin   non-polarized
Spin.Fix   false
Spin.Total   0

%block kgrid_Monkhorst_Pack
  30   0   0   0.0
  0   30   0   0.0
  0   0   1   0.0
%endblock kgrid_Monkhorst_Pack
