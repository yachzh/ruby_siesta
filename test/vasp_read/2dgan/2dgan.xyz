    2

Ga      0.000000    0.000000   20.000000
N       0.000000    1.857913   20.000000
