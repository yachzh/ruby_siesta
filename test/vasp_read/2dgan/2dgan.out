Siesta Version  : 4.1.5
Architecture    : arm64
Compiler version: GNU Fortran (Homebrew GCC 12.2.0) 12.2.0
Compiler flags  : mpif90 -O2 -fPIC -ftree-vectorize -fallow-argument-mismatch
PP flags        : -DCDF -DNCDF -DNCDF_4 -DMPI -DFC_HAVE_ABORT
Libraries       : libncdf.a libfdict.a -L/opt/homebrew/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5  -lz -lscalapack -L/opt/homebrew/opt/openblas/lib -lopenblas
PARALLEL version
NetCDF support
NetCDF-4 support

* Running on 4 nodes in parallel
>> Start of run:  24-JUL-2024  21:14:18

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from 2dgan.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: 2dgan
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   31 Label: Ga
Species number:   2 Atomic number:    7 Label: N

Ground state valence configuration:   4s02  4p01
Reading pseudopotential information in formatted form from Ga.psf

Valence configuration for pseudopotential generation:
4s( 2.00) rc: 2.18
4p( 1.00) rc: 2.47
4d( 0.00) rc: 2.56
4f( 0.00) rc: 2.56
Ground state valence configuration:   2s02  2p03
Reading pseudopotential information in formatted form from N.psf

Valence configuration for pseudopotential generation:
2s( 2.00) rc: 1.48
2p( 3.00) rc: 1.48
3d( 0.00) rc: 1.48
4f( 0.00) rc: 1.48
For Ga, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For N, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Ga                   Z=  31    Mass=  69.720        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=4
          i=1  nzeta=2  polorb=0  (4s)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=4
          i=1  nzeta=2  polorb=1  (4p)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for Ga                    (Z =  31)

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge for ps generation:    3.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  3.4161
V l=1 = -2*Zval/r beyond r=  3.4161
V l=2 = -2*Zval/r beyond r=  3.4161
V l=3 = -2*Zval/r beyond r=  3.4161
All V_l potentials equal beyond r=  2.5307
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  3.4161
Using large-core scheme for Vlocal

atom: Estimated core radius    3.41607
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    4.01884
atom: Maximum radius for r*vlocal+2*Zval:    3.45904
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  2.627367   el= -0.640168   Ekb=  4.978370   kbcos=  0.179028
   l= 1   rc=  2.627367   el= -0.192284   Ekb=  1.535068   kbcos=  0.159995
   l= 2   rc=  2.627367   el=  0.002051   Ekb= 26.910316   kbcos=  0.007797
   l= 3   rc=  2.627367   el=  0.003185   Ekb= -0.922651   kbcos= -0.014048

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    6.625996
                 energy =   -0.637061
                kinetic =    0.438116
    potential(screened) =   -1.075178
       potential(ionic) =   -2.594980

   izeta = 2
                 rmatch =    4.970382
              splitnorm =    0.150000
                 energy =   -0.558271
                kinetic =    0.703210
    potential(screened) =   -1.261481
       potential(ionic) =   -2.845290

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 4p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    9.170612
                 energy =   -0.188622
                kinetic =    0.553474
    potential(screened) =   -0.742096
       potential(ionic) =   -2.078924

   izeta = 2
                 rmatch =    6.462398
              splitnorm =    0.150000
                 energy =   -0.141220
                kinetic =    0.799388
    potential(screened) =   -0.940608
       potential(ionic) =   -2.392431

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 4p

   izeta = 1
                     rc =    9.170612
                 energy =    0.249710
                kinetic =    0.755806
    potential(screened) =   -0.506097
       potential(ionic) =   -1.702942
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 4s( 2.00)                                                            
 4p( 1.00)                                                            
Vna: chval, zval:    3.00000   3.00000

Vna:  Cut-off radius for the neutral-atom potential:   9.170612
comcore: Pseudo-core radius Rcore=  5.096210

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
N                    Z=   7    Mass=  14.010        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=1  (2p)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for N                     (Z =   7)

read_vps: Pseudopotential generation method:
read_vps: ATM 3.2.2 Troullier-Martins                       
Valence charge for ps generation:    5.00000

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  1.4245
V l=1 = -2*Zval/r beyond r=  1.4068
V l=2 = -2*Zval/r beyond r=  1.4068
V l=3 = -2*Zval/r beyond r=  1.4068
All V_l potentials equal beyond r=  1.4068
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.4245

VLOCAL1: 99.0% of the norm of Vloc inside     21.932 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     49.984 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    1.69703
atom: Maximum radius for r*vlocal+2*Zval:    1.47898
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  1.574385   el= -1.365823   Ekb=  5.514362   kbcos=  0.335894
   l= 1   rc=  1.554824   el= -0.521103   Ekb= -5.603257   kbcos= -0.412958
   l= 2   rc=  1.655124   el=  0.001990   Ekb= -1.308906   kbcos= -0.006857
   l= 3   rc=  1.697033   el=  0.003093   Ekb= -0.526601   kbcos= -0.000860

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    4.388595
                 energy =   -1.362128
                kinetic =    1.166067
    potential(screened) =   -2.528195
       potential(ionic) =   -8.067871

   izeta = 2
                 rmatch =    2.941647
              splitnorm =    0.150000
                 energy =   -1.185279
                kinetic =    1.773691
    potential(screened) =   -2.958970
       potential(ionic) =   -8.905783

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    5.496027
                 energy =   -0.517524
                kinetic =    3.419844
    potential(screened) =   -3.937368
       potential(ionic) =   -9.259482

   izeta = 2
                 rmatch =    3.092486
              splitnorm =    0.150000
                 energy =   -0.339249
                kinetic =    5.062824
    potential(screened) =   -5.402073
       potential(ionic) =  -11.346871

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p

   izeta = 1
                     rc =    5.496027
                 energy =    1.497507
                kinetic =    3.055801
    potential(screened) =   -1.558295
       potential(ionic) =   -5.761665
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 3.00)                                                            
Vna: chval, zval:    5.00000   5.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.496027

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   31 Ga                      # Species index, atomic number, species label
    2    7 N                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
Ga                    2                    # Species label, number of l-shells
 n=4   0   2                         # n, l, Nzeta 
   6.626      4.970   
   1.000      1.000   
 n=4   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   9.171      6.462   
   1.000      1.000   
N                     2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   4.389      2.942   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.496      3.092   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Ga.ion.nc
Dumping basis to NetCDF file N.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000  37.79454  1        1
siesta:      0.00000   3.51095  37.79454  2        2

siesta: System type = slab      

initatomlists: Number of atoms, orbitals, and projectors:      2    26    32

coxmol: Writing XMOL coordinates into file 2dgan.xyz                                                           

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   200.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     2000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   999.9927 K
redata: Fix the spin of the system                  =   F
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 5
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 5
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000

* ProcessorY, Blocksize:    2   7


* Orbital distribution balance (max,min):     7     5

k-point displ. along   1 input, could be:     0.00    0.50
k-point displ. along   2 input, could be:     0.00    0.50
 Kpoints in:          480 . Kpoints trimmed:          466

siesta: k-grid: Number of k-points =   466
siesta: k-grid: Cutoff (effective) =    20.000 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:   30   0   0      0.000
siesta: k-grid:    0  30   0      0.000
siesta: k-grid:    0   0   1      0.000

diag: Algorithm                                     = D&C
diag: Parallel over k                               =   F
diag: Use parallel 2D distribution                  =   T
diag: Parallel block-size                           = 7
diag: Parallel distribution                         =     2 x     2
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     9 x     9 x     1  =      81
superc: Number of atoms, orbitals, and projectors:    162   2106   2592


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     9 x     9 x     1  =      81
superc: Number of atoms, orbitals, and projectors:    162   2106   2592

outcell: Unit cell vectors (Ang):
        3.218000    0.000000    0.000000
       -1.609000    2.786870    0.000000
        0.000000    0.000000   40.000000

outcell: Cell vector modules (Ang)   :    3.218000    3.218000   40.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000    120.0000
outcell: Cell volume (Ang**3)        :    358.7259
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=7 sparsity=7.0355 nnzs=4756, refcount: 7>
  <dData1D:(new from dSpData1D) n=4756, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=7 sparsity=7.0355 nnzs=4756, refcount: 8>
  <dData2D:DM n=4756 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      25     202
New grid distribution:   1
           1       1:   12    1:    6    1:   90
           2       1:   12    1:    6   91:  180
           3       1:   12    7:   12    1:   90
           4       1:   12    7:   12   91:  180

InitMesh: MESH =    24 x    24 x   360 =      207360
InitMesh: (bp) =    12 x    12 x   180 =       25920
InitMesh: Mesh cutoff (required, used) =   200.000   204.970 Ry
ExtMesh (bp) on 0 =    96 x    90 x   178 =     1537920
New grid distribution:   2
           1       1:   12    1:   12    1:   85
           2       1:   12    1:   12   96:  180
           3       1:   12    1:   12   86:   90
           4       1:   12    1:   12   91:   95
New grid distribution:   3
           1       1:   12    1:   12    1:   79
           2       1:   12    1:   12  102:  180
           3       1:   12    1:   12   80:   90
           4       1:   12    1:   12   91:  101
Setting up quadratic distribution...
ExtMesh (bp) on 0 =    96 x    96 x   173 =     1594368
PhiOnMesh: Number of (b)points on node 0 =                12240
PhiOnMesh: nlist on node 0 =               126521

stepf: Methfessel-Paxton step function
       Using Hermite-Gauss polynomials of order  4

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -32.388824
siesta: Eions   =       510.813266
siesta: Ena     =        93.237765
siesta: Ekin    =       253.444475
siesta: Enl     =       -37.603089
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -27.228625
siesta: DUscf   =         7.057951
siesta: DUext   =         0.000000
siesta: Exc     =      -103.641253
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -315.763502
siesta: Etot    =      -325.546041
siesta: FreeEng =      -325.546041

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -315.763502     -325.546041     -325.546041  1.787073 -7.167175 13.443076
timer: Routine,Calls,Time,% = IterSCF        1       1.527  52.53
   scf:    2     -325.994415     -325.771398     -325.771398  0.007134 -7.077743 13.051277
   scf:    3     -329.619734     -329.324367     -329.324367  0.221252 -4.140979  1.388664
   scf:    4     -329.092698     -329.311546     -329.311546  0.044992 -4.804099  2.223476
   scf:    5     -329.377836     -329.346281     -329.346281  0.038260 -4.452019  1.484884
   scf:    6     -329.407005     -329.392740     -329.392740  0.016646 -4.134860  0.015197
   scf:    7     -329.392598     -329.392695     -329.392695  0.000657 -4.121490  0.050586
   scf:    8     -329.392776     -329.392749     -329.392749  0.003367 -4.126797  0.005889
   scf:    9     -329.392750     -329.392749     -329.392749  0.000151 -4.127931  0.002806
   scf:   10     -329.392750     -329.392750     -329.392750  0.000015 -4.128017  0.002501
   scf:   11     -329.392750     -329.392750     -329.392750  0.000065 -4.128874  0.000017

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000654689
max |H_out - H_in|      (eV) :     0.0000173064
SCF cycle converged after 11 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      25     202

siesta: E_KS(eV) =             -329.3927

siesta: E_KS - E_eggbox =      -329.3927

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000049    0.000028    0.000000
----------------------------------------
   Max    0.000352
   Res    0.000219    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000352    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -74.96      -74.98        0.02       -0.00        0.00       -0.01
(Free)E + p*V (eV/cell)     -318.2040
Target enthalpy (eV/cell)     -329.3927

mulliken: Atomic and Orbital Populations:

Species: Ga                  
Atom  Qatom  Qorb
               4s      4s      4py     4pz     4px     4py     4pz     4px     
               4Pdxy   4Pdyz   4Pdz2   4Pdxz   4Pdx2-y2
   1  2.435   0.512   0.442   0.119   0.243   0.119   0.401   0.155   0.401
              0.018   0.014  -0.020   0.014   0.018

Species: N                   
Atom  Qatom  Qorb
               2s      2s      2py     2pz     2px     2py     2pz     2px     
               3Pdxy   3Pdyz   3Pdz2   3Pdxz   3Pdx2-y2
   2  5.565   1.121   0.343   1.111   1.637   1.111   0.137  -0.071   0.137
              0.014   0.004   0.002   0.004   0.014

mulliken: Qtot =        8.000

coxmol: Writing XMOL coordinates into file 2dgan.xyz                                                           

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -83.819128
siesta: Eions   =       510.813266
siesta: Ena     =        93.237765
siesta: Ekin    =       211.867264
siesta: Enl     =       -24.658249
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -4.703655
siesta: DUscf   =         0.918206
siesta: DUext   =         0.000000
siesta: Exc     =       -95.240815
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -329.392750
siesta: Etot    =      -329.392750
siesta: FreeEng =      -329.392750

siesta: Final energy (eV):
siesta:  Band Struct. =     -83.819128
siesta:       Kinetic =     211.867264
siesta:       Hartree =    2003.686125
siesta:       Edftu   =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =     -95.240815
siesta:  Ion-electron =   -4184.937140
siesta:       Ion-ion =    1735.231817
siesta:       Ekinion =       0.000000
siesta:         Total =    -329.392750
siesta:         Fermi =      -4.128874

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.046787   -0.000008   -0.000000
siesta:    -0.000008   -0.046796    0.000000
siesta:     0.000000   -0.000000    0.000013

siesta: Cell volume =        358.725874 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00033970          0.00033970  Ry/Bohr**3
siesta:           0.03119031          0.03118998  eV/Ang**3
siesta:          49.97292069         49.97239908  kBar
(Free)E+ p_basis*V_orbitals  =        -327.302517
(Free)Eharris+ p_basis*V_orbitals  =        -327.302517

siesta: Electric dipole (a.u.)  =   -0.000000   -0.000000   -0.000000
siesta: Electric dipole (Debye) =   -0.000000   -0.000000   -0.000000

cite: Please see "2dgan.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 4.1.5
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302

>> End of run:  24-JUL-2024  21:14:33
Job completed
